### PR TITLE
feat(models): add sortie & events

### PR DESCRIPTION
**Pull request by Mettwasser**

### CODE REVIEW

**Review by Mettwasser — COMMENTED** [review]
gud

**Comment by Mettwasser** [comment]
Good, couldn't notice something faulty/wrong

**Review by Mettwasser — COMMENTED** [review]
Good, couldn't notice something faulty/wrong

File: warframe/worldstate/enums/faction.py [hunk 1]
```diff
@@ -11,3 +11,4 @@ class Faction(Enum):
     Grineer = "Grineer"
     Tenno = "Tenno"
     Narmer = "Narmer"
+    Infestation = "Infestation"
```
Comment (TobiTenno) [inline warframe/worldstate/enums/faction.py]: Infested? not seen an infestation display in a while

File: warframe/worldstate/models/event.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+from ..common import MultiQueryModel
+from ..enums import MissionType, Faction, Syndicate
+
+__all__ = ["Event"]
+
+
+class Event(MultiQueryModel):
+    activation: Optional[datetime] = None
+    "The time the Event started"
```
Comment (TobiTenno) [inline warframe/worldstate/models/event.py:16]: it's your code, but i don't love the members being completely unspaced with the docstrings...

**Review by TobiTenno — CHANGES_REQUESTED** [review]

**Review by TobiTenno — APPROVED** [review]

**Pull request merged**